board/common/rootfs/libexec/infix/init.d/99-done

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -s /run/finit/cond/reconf /run/finit/cond/usr/ixinit-done
```

Comment:
Setting conditions with `initctl` is possible early in runlevel S, and may be a cleaner way of doing this.  For an example, see:

https://github.com/kernelkit/infix/blob/91148de095b201d2ee800bfe261ac34228ea951f/board/common/rootfs/libexec/infix/mkcert#L40

Which should be `initctl -bq ...`, I'm realizing now.

Reply:
TIL, thanks! :+1: